### DISCUSSION TITLE
Insomnia doesn't allow altering the Accept header any more?

### DISCUSSION THREAD

**Discussion by rehevkor5** [Kong/insomnia#8260] [Ideas]
Why is the Accept header hard-coded to `*/*`? I should be able to specify other values, such as `application/json`.

**Comment by subnetmarco** [comment]
You can easily overwrite it:

![Screenshot 2024-12-23 at 8 46 38 AM](https://github.com/user-attachments/assets/46eb0df9-1bd0-4b51-833d-99997b34e67b)

**Reply by Dudeplayz** [reply]
I think this is confusing. The default one should either be removed or visually be marked, maybe line-through?

**Reply by The-Futurist** [reply]
> You can easily overwrite it:
> 
> ![Screenshot 2024-12-23 at 8 46 38 AM](https://private-user-images.githubusercontent.com/964813/398175125-46eb0df9-1bd0-4b51-833d-99997b34e67b.png?jwt=eyJ0eXAiOiJKV1QiLCJhbGciOiJIUzI1NiJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3NjIyODQwMzIsIm5iZiI6MTc2MjI4MzczMiwicGF0aCI6Ii85NjQ4MTMvMzk4MTc1MTI1LTQ2ZWIwZGY5LTFiZDAtNGI1MS04MzNkLTk5OTk3YjM0ZTY3Yi5wbmc_WC1BbXotQWxnb3JpdGhtPUFXUzQtSE1BQy1TSEEyNTYmWC1BbXotQ3JlZGVudGlhbD1BS0lBVkNPRFlMU0E1M1BRSzRaQSUyRjIwMjUxMTA0JTJGdXMtZWFzdC0xJTJGczMlMkZhd3M0X3JlcXVlc3QmWC1BbXotRGF0ZT0yMDI1MTEwNFQxOTE1MzJaJlgtQW16LUV4cGlyZXM9MzAwJlgtQW16LVNpZ25hdHVyZT0xOGNiNDIzNGVkNDJhNjEwNDI3OWNiZTA3MmYzMjZlYTdiNTQ2Njk3YmIwOTYwNjNiNDRkOTEyOTU4MDA5YWY0JlgtQW16LVNpZ25lZEhlYWRlcnM9aG9zdCJ9.hNzI1eEt0qbeacwY7agFMDmrIzO-zKfnzt8kJoumkd4)

Do you mean that when we add a _second_ Accept header, it uses that and does not send the first? If so that's good to know but far from obvious.

**Reply by ppazos** [reply]
@subnetmarco this is indeed confusing, at least visually.

**Reply by subnetmarco** [reply]
@chayglass84
